app/lib/actions/pulp/consumer/sync_node.rb

```diff
@@ -9,6 +9,29 @@ class SyncNode < AbstractSyncNodeTask
         end
 
         def invoke_external_task
+          task = invoke_external_task_based_on_repos_id
+          schedule_timeout(Setting['content_action_accept_timeout'])
+          task
+        end
+
+        def process_timeout
+          accept_timeout = Setting['content_action_accept_timeout']
+          finish_timeout = Setting['content_action_finish_timeout']
```

Comment:
We may want to use a different timeout from the existing content_action_finish_timeout.  My reasoning is that a system installing some packages should not take longer than an hour under *most* circumstances.  The node sync hoewver probably will take longer to sync than an hour for most people on the initial sync.  I think its fine to reuse the accept_timeout though.